vignettes/mmrm_review_main.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "Working title for `mmrm` review and comparison"
+package: mmrm
+bibliography: '`r system.file("REFERENCES.bib", package = "mmrm")`'
+output:
+  rmarkdown::html_document:
+          theme: "spacelab"
+          highlight: "kate"
+          toc: true
+          toc_float: true
+vignette: |
+  %\VignetteIndexEntry{Working title for `mmrm` review and comparison}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+# Abstract
+
+```{r, child=c('mmrm_review_abstract.Rmd')}
+```
+
+# Introduction
+
+Ideas: 
+A general, textbook-like, introduction which outlines MMRM and how it works with a 
+focus on key references. 
+It will provide s plainly written description, and frame mmrm in contrast to 
+random slope intercept / glm / ... model. 
+Why do we use mmrm rather than other models. 
+data types, covariance structures, existing implementations in R and SAS proc mixed
+misnomer of "mixed model", since we are actually marginalizing out the random effects
+
+```{r, child=c('mmrm_review_introduction.Rmd')}
+```
+
+# Methods
+
+introduction of data sets, need to consider what type of data (simulated or real) 
+data problems (low number of observations, varying scales of variables) 
+could be quite rigorous.
+should at least illustrate that SAS/R are producing similar results on the FEV 
```

Comment:
agree with Daniel that it would be more accessible to split the "intro" (theoretical background, scoping, naming) from the comparison with SAS

Reply:
yes, this should be sensibly structured

Reply:
Yeah please into two vignettes